src/views/v2/month/calendar-body/day/multiday-events/multiday-event.php

```diff
@@ -29,62 +29,89 @@
+		$multiday_classes[] = 'tribe-events-calendar-month__multiday-event--start';
 	}
 
 	// If it ends this week, let's add the end class (right border).
 	if ( $event->ends_this_week ) {
-		$classes[] = 'tribe-events-calendar-month__multiday-event--end';
+		$multiday_classes[] = 'tribe-events-calendar-month__multiday-event--end';
 	}
 
 	if ( $event->dates->end->format( 'Y-m-d' ) < $today_date ) {
-		$classes[] = 'tribe-events-calendar-month__multiday-event--past';
+		$multiday_classes[] = 'tribe-events-calendar-month__multiday-event--past';
 	}
 
-
-} else {
-	$classes[] = 'tribe-events-calendar-month__multiday-event--hidden';
 }
 
 ?>
 <div class="tribe-events-calendar-month__multiday-event-wrapper">
-	<article <?php tribe_classes( $classes ); ?> data-event-id="<?php echo esc_attr( $event->ID ); ?>">
-		<time
-			datetime="<?php echo esc_attr( $event->dates->start->format( Dates::DBDATEFORMAT ) ); ?>"
-			class="tribe-common-a11y-visual-hide"
-		>
-			<?php echo esc_attr( $event->dates->start->format( Dates::DBDATEFORMAT ) ); ?>
-		</time>
-		<a href="<?php echo esc_url( $event->permalink ) ?>" class="tribe-events-calendar-month__multiday-event-inner">
-			<?php if ( $event->featured ) : ?>
-				<em
-					class="tribe-events-calendar-month__multiday-event-featured-icon tribe-common-svgicon tribe-common-svgicon--featured"
-					aria-label="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
-					title="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
-				></em>
-			<?php endif; ?>
-			<h3 class="tribe-events-calendar-month__multiday-event-title tribe-common-h8">
-				<?php echo get_the_title( $event ); ?>
-			</h3>
-		</a>
+	<article <?php tribe_classes( $event_classes ); ?> data-event-id="<?php echo esc_attr( $event->ID ); ?>">
+		<div class="tribe-events-calendar-month__multiday-event--hidden">
+			<time
+				datetime="<?php echo esc_attr( $event->dates->start->format( Dates::DBDATEFORMAT ) ); ?>"
+				class="tribe-common-a11y-visual-hide"
+			>
+				<?php echo esc_attr( $event->dates->start->format( Dates::DBDATEFORMAT ) ); ?>
+			</time>
+			<a
+				href="<?php echo esc_url( $event->permalink ) ?>"
+				<?php if ( $display_tooltip ) : ?>
+					data-js="tribe-events-tooltip"
+					data-tooltip-content="#tribe-events-tooltip-content-<?php echo esc_attr( $event->ID ); ?>"
+					aria-describedby="tribe-events-tooltip-content-<?php echo esc_attr( $event->ID ); ?>"
+				<?php endif; ?>
+			>
+				<?php if ( $event->featured ) : ?>
+					<em
+						class="tribe-events-calendar-month__multiday-event-featured-icon tribe-common-svgicon tribe-common-svgicon--featured"
+						aria-label="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
+						title="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
+					></em>
+				<?php endif; ?>
+				<h3 class="tribe-events-calendar-month__multiday-event-title tribe-common-h8">
+					<?php echo get_the_title( $event ); ?>
```

Comment:
:no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'get_the_title'.

<sub>hash: 8b14ee1b6ff0fd80b1e99d7723cfffcd</sub>